less/current/base/scaffolding.less

```diff
@@ -2,7 +2,7 @@
 // Scaffolding
 // --------------------------------------------------
 
-* {
```

Comment:
Using * selector means that font-size, font-family and line-height overrides need to happen in the same element and cannot leverage cascading from the parent element, so using html, body selector is more flexible

I checked that this does not impact the appearance of any of the public or private pages